space-cli/cmd/modules/ingress/getters_test.go

```diff
@@ -64,6 +65,313 @@ func TestGetIngressRoutes(t *testing.T) {
 		},
+		{
+			name: "Url Filter Passes",
+			args: args{
+				project:     "myproject",
+				commandName: "ingress-routes",
+				params:      map[string]string{},
+				filters:     []string{"url=/v1/config/projects/myproject/routing/ingress"},
+			},
+			transportMockArgs: []mockArgs{
+				{
+					method: "Get",
+					args:   []interface{}{"GET", "/v1/config/projects/myproject/routing/ingress", map[string]string{}, new(model.Response)},
+					paramsReturned: []interface{}{nil, model.Response{
+						Result: []interface{}{map[string]interface{}{
+							"id": "local-admin",
+							"source": map[string]interface{}{
+								"url": "/v1/config/projects/myproject/routing/ingress",
+							},
+							"targets": map[string]interface{}{
+								"version": "v0.18.0",
+							},
+						},
+						},
+					}},
+				},
+			},
+			want: []*model.SpecObject{
+				{
+					API:  "/v1/config/projects/{project}/routing/ingress/{id}",
+					Type: "ingress-routes",
+					Meta: map[string]string{"project": "myproject", "id": "local-admin"},
+					Spec: map[string]interface{}{"source": map[string]interface{}{"url": "/v1/config/projects/myproject/routing/ingress"}, "targets": map[string]interface{}{"version": "v0.18.0"}},
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "Url Filter Fails",
+			args: args{
+				project:     "myproject",
+				commandName: "ingress-routes",
+				params:      map[string]string{},
+				filters:     []string{"url=/v1/config/projects/myproject/routing"},
+			},
+			transportMockArgs: []mockArgs{
+				{
+					method: "Get",
+					args:   []interface{}{"GET", "/v1/config/projects/myproject/routing/ingress", map[string]string{}, new(model.Response)},
+					paramsReturned: []interface{}{nil, model.Response{
+						Result: []interface{}{map[string]interface{}{
+							"id": "local-admin",
+							"source": map[string]interface{}{
+								"url": "/v1/config/projects/myproject/routing/ingress",
+							},
+							"targets": map[string]interface{}{
+								"version": "v0.18.0",
+							},
+						},
+						},
+					}},
+				},
+			},
+			want:    []*model.SpecObject{},
+			wantErr: false,
+		},
+		{
+			name: "Service Filter Passes",
+			args: args{
+				project:     "myproject",
+				commandName: "ingress-routes",
+				params:      map[string]string{},
+				filters:     []string{"service=greeting"},
+			},
+			transportMockArgs: []mockArgs{
+				{
+					method: "Get",
+					args:   []interface{}{"GET", "/v1/config/projects/myproject/routing/ingress", map[string]string{}, new(model.Response)},
+					paramsReturned: []interface{}{nil, model.Response{
+						Result: []interface{}{map[string]interface{}{
+							"id": "local-admin",
+							"source": map[string]interface{}{
+								"url": "/v1/config/projects/myproject/routing/ingress",
+							},
+							"targets": []interface{}{
+								map[string]interface{}{
+									"version": "v0.18.0",
+									"host":    "greeting.myproject.svc.cluster.local",
+								},
+							},
```

Comment:
Keep two same targets.
```suggestion
							"targets": []interface{}{
								map[string]interface{}{
									"version": "v0.18.0",
									"host":    "greeting.myproject.svc.cluster.local",
								},
								map[string]interface{}{
									"version": "v0.18.0",
									"host":    "greeting.myproject.svc.cluster.local",
								},
							},
```